src/StackHCIVM/StackHCIVM.Autorest/test/AzStackHCIVMImageNew.Tests.ps1

```diff
@@ -15,13 +15,9 @@ if(($null -eq $TestName) -or ($TestName -contains 'AzStackHCIVMImageNew'))
 }
 
 Describe 'AzStackHCIVMImageNew' {
-    It 'Create Image '  {
```

Comment:
The modifications made in the "test" section appear unrelated to the fix. Please include a test specifically addressing the issue fix in the "Add-AzStackHCIVMVirtualMachineDataDisk" function and remove unneeded test changes.

Reply:
Undid this. Most of the test changes are related to modifying names because the old rg doesnt exist anymore.